performance-tests/src/models/token.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+export interface TokenResponse {
```

Comment:
Should be called `JwtResponse`. `Token` is misleading because there are Tokens in enmeshed.